app/assets/stylesheets/mobile/header.scss

```diff
@@ -118,16 +136,16 @@ header {
       line-height: 25px;
       font-weight: bold;
       color: $light-grey;
-      border-bottom: solid rgb(53, 53, 53) 2px;
+      border-bottom: solid $light-grey 1px;
```

Comment:
light-grey is #ddd, looks like you did a mistake here. And indeed, the borders are #ddd so almost white for me, not #353535 as actually (so your screenshot is not up-to-date). This color doesn't exist in our variables currently. $border-dark-grey is #999 which is still to light imo. Maybe it's worth creating a new variable?